libs/media-messages/src/generated/types/payloads/media/commands/delete-entity-command.ts

```diff
@@ -19,8 +18,5 @@ export interface DeleteEntityCommand {
    * The name of the entity type.
    */
   entity_type?: string;
-  /**
-   * Additional input data.
-   */
-  input?: BulkEntityInput;
+  input?: { [name: string]: unknown };
```

Comment:
The description got lost here and with the externalized type (and the description there) it was much clearer what this is used for.

Reply:
Related to Ruwans comment: https://github.com/Axinom/mosaic-media-template/pull/98#pullrequestreview-1500138990